firefox-ios/firefox-ios-tests/Tests/XCUITests/FindInPageTests.swift

```diff
@@ -46,30 +51,44 @@ class FindInPageTests: BaseTestCase {
         openFindInPageFromMenu(openSite: userState.url!)
 
         // Enter some text to start finding
-        app.searchFields["find.searchField"].typeText("Book")
+        if #available(iOS 16, *) {
+            app.searchFields["find.searchField"].typeText("Book")
+        } else {
+            app.textFields["FindInPage.searchField"].typeText("Book")
+        }
 
         // Once there are matches, test previous/next buttons
-        mozWaitForElementToExist(app.staticTexts["1 of 6"])
-        XCTAssertTrue(app.staticTexts["1 of 6"].exists)
+        if #available(iOS 16, *) {
```

Comment:
Texts such as "1/6", "2/6" etc during the searches are not visible via `print(app.debugDescription)` for iOS 15 only. Still, we should exercise the buttons instead of disabling the whole test.